Add Loading component (DL-10)

r0o0 opened the pull request.

r0o0 committed:
Fix styleguidist components directory

r0o0 committed:
Add Loading component

r0o0 committed:
Add enzyme and configure adapter to jest

r0o0 committed:
Configure styleguide config to use react-script webpack config

r0o0 committed:
Add downlevelIteration setting in tsconfig

codemacaron commented on client/src/components/Loading/components/Spinner/Spinner.scss:
```diff
@@ -0,0 +1,23 @@
+
+.spinner {
+  width: 20px;
+  height: 20px;
```
spinner size를 대 중 소로 가능 하도록 작업 해주세요.

r0o0 replied:
좋은 아이디어 입니당! ㅎㅎ
'small' | 'medium' | 'large' 사이즈로  enum 등록하고 (47f860e) 수정했어요~ 6c75ff0

codemacaron commented on client/src/components/Loading/Loading.tsx:
```diff
@@ -0,0 +1,33 @@
+
+  return (
+    <span {...rest} className={loadingClass} role="status" aria-label={children != null ? '로딩중' : undefined}>
+      {type === LOADING_TYPE.TEXT && <Dots />}
```
text와 함께 사이즈를 대 중 소로 만들어 주세요.

r0o0 replied:
text font-size에 맞게 dots 사이즈 알맞게 추가했어요. 6c75ff0

r0o0 committed:
Add font styles

- Add Nanmu Square font-family for Kor lang
- Add Open Sann font-family for En lang

r0o0 committed:
Add Size enum type

r0o0 committed:
Fix by review

r0o0 committed:
Fix StyleContainer default direction from horizontal to vertical

r0o0 commented:
StyleContainer의 direction 값이 default로 `horizontal`로 되어있더라구요;; 앞으로 이 컴포넌트 사용할대 햇갈릴것 같아서 `vertical`로 수정했어요. 이부분 수정하면서 Loading.md에 StyleContainter 쓸 수 있는 부분들도 수정했어요. 14ccd16
